framework/client/form.rst

```diff
@@ -0,0 +1,155 @@
+Form
+=====
+
+formField
+----------
+
+1. editableValueFactory
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+  editableValueFactory allows to pass the type attributes of the different fields for all fields from one place. It
+  provides the field with the requested props and gets the data from formField through the parent formField
+  FormDataContainer (point 7).
+
+  It returns the component ``<EditableValue>`` with these props:
+
+  * type
+  * value
+  * info
+  * events
+  * options
+
+2. formattedValueFactory
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  Analog to editableValueFactory for formatted values with only two props type and value.
+
+3. defaultMapping
+~~~~~~~~~~~~~~~~~~
+
+  Here the types of form fields get mapped and exported as an object consisting of type string
+  attributes and components as values. In this case values are often a call to editableValueFactory.
+  This way it’s possible to map the same call to editableValueFactory multiple times and provide the same
+  field to different types.
+
+  Example:
+
+  .. code:: js
+
+    ‘longitude’: editableValueFactory(‘coordinate’),
+    ‘latitude’: editableValueFactory(‘coordinate’)
+
+
+4. FormFieldFactory
+~~~~~~~~~~~~~~~~~~~~
+
+  FormFieldFactory receives the data for identifying the correct field from the redux-store through a redux container and
+  provides it to ``<ValueField/>``. The react-redux container is FormDataContainer (point 9). ``<ValueField/>`` returns the specific field
+  accessed through the mapping of the given type of the field.
+
+5. typeEditables
+~~~~~~~~~~~~~~~~~
+
+  The typeEditables directory contains files for different field types like ‘remote’. The files contain methods to overwrite
+  the default behaviour. This can be done through methods that get the events, value and options or data for
```

Comment:
The files contain methods to overwrite
  the default behaviour `for each type`.